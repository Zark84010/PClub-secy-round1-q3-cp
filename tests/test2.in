10 9987 8
10528 21924 32550 9702 3332 10829 7140 2261 2261 2261 
c 1 459
f 3 3
f 2 10
c 3 187
f 3 6
f 8 9
c 3 323
f 10 10
