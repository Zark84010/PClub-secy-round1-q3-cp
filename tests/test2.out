324946650
69909
9987
22580607
22580607
